integration-tests/api/__tests__/returns/ff-tax-inclusive-pricing.js

```diff
@@ -0,0 +1,175 @@
+const path = require("path")
+
+const startServerWithEnvironment =
+  require("../../../helpers/start-server-with-environment").default
+const { useApi } = require("../../../helpers/use-api")
+const { useDb } = require("../../../helpers/use-db")
+
+const { simpleProductFactory, simpleOrderFactory } = require("../../factories")
+const adminSeeder = require("../../helpers/admin-seeder")
+
+const createReturnableOrder = async (dbConnection, options) => {
+  await simpleProductFactory(
+    dbConnection,
+    {
+      id: "test-product",
+      variants: [{ id: "test-variant" }],
+    },
+    100
+  )
+
+  let discounts = []
+
+  if (options?.discount) {
+    discounts = [
+      {
+        code: "TESTCODE",
+      },
+    ]
+  }
+
+  return await simpleOrderFactory(dbConnection, {
+    email: "test@testson.com",
+    tax_rate: options?.oldTaxes ? undefined : null,
+    region: {
+      id: "test-region",
+      name: "Test region",
+      tax_rate: 12.5,
+    },
+    discounts,
+    line_items: [
+      {
+        id: "test-item",
+        variant_id: "test-variant",
+        quantity: 2,
+        fulfilled_quantity: options?.shipped ? 2 : undefined,
+        shipped_quantity: options?.shipped ? 2 : undefined,
+        unit_price: options?.oldTaxes ? 1125 : 1200,
+        includes_tax: true,
```

Comment:
suggestion(non-blocking): maybe add 1-2 tests where `includes_tax: false` to cover that as well?

Reply:
Yeah sure

Reply:
done